### ISSUE TITLE
Esperanto fails to bundle with ES6 modules that call super()

### ISSUE THREAD

**Issue by jvdl** [esperantojs/esperanto#129]
in esperanto 0.6.18, bundling a class that calls super will fail (both Acorn and Estraverse were upgraded - not ideal to do this in a patch release IMO).

I've put together a reduced test case: https://bitbucket.org/snippets/jvanderloo/Mdyy

(you can git clone that locally and `npm install && node index.js` to see the bug in action)

Relevant code:
```js
esperanto.bundle({
    entry: 'foo.js'
}).then(function (bundle) {
    console.log(bundle.toCjs().code);
}).catch(function (err) {
    console.error(err);
});
```

Expected output: "esperanto-ified" contents of foo.js

Actual output: `[Error: Unknown node type Super.]`

**Comment by Rich-Harris** [comment]
Ah, whoops. The decision to do a patch release was justified on *paper* (it was a bugfix release; the use of different dependency versions is just an implementation detail) but in retrospect it was perhaps a bit bold. Are you able to continue using 0.6.17 until I can get this fixed?

Does anyone know of any good alternatives to estraverse that are up to date? Or should we just roll our own?

**Comment by Rich-Harris** [comment]
@jvdl This is fixed in 0.6.21 (I figured it had to be another patch release, otherwise people using ^0.6 would be stuck with the buggy version!). I've removed estraverse in favour of an internal utility that doesn't depend on hardcoded node types, and therefore isn't subject to the same thing happening again the next time the ESTree spec is revised. Can confirm it fixes your reduced test case (thanks for providing it)

**Comment by jvdl** [comment]
Thanks for fixing @Rich-Harris. We were indeed depending on ^0.6. Can confirm that it's working now :smile:
